### PR TITLE
[Behat][API] Implement iri converter providing iri based on a given section

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | api-platform-3|
| Bug fix?        | yes                                                       |
| New feature?    | yes                                                       |
| BC breaks?      | no                                                      |
| Deprecations?   | no |
| Related tickets | |
| License         | MIT                                                          |

### CODE REVIEW

File: src/Sylius/Behat/Context/Api/Admin/ManagingCatalogPromotionsContext.php [hunk 1]
```diff
@@ -144,7 +142,7 @@ public function iDescribeItAsIn(string $description, string $localeCode): void
      */
     public function iMakeItAvailableInChannel(ChannelInterface $channel): void
     {
-        $this->client->addRequestData('channels', [$this->sectionAwareIriConverter->getIriFromResourceInSection($channel, 'admin')]);
+        $this->client->addRequestData('channels', [$this->iriConverter->getIriFromResource($channel)]);
```
Comment (Wojdylak) [inline src/Sylius/Behat/Context/Api/Admin/ManagingCatalogPromotionsContext.php]: Why did you remove section `admin`?

**Review by Wojdylak — APPROVED** [review]

**Review by Wojdylak — APPROVED** [review]

**Review by NoResponseMate — APPROVED** [review]

**Review by TheMilek — APPROVED** [review]

**Comment by TheMilek** [comment]
Thank you @GSadee! 🥇
